desktop_version/src/main.cpp

```diff
@@ -1,6 +1,8 @@
 #include <SDL.h>
 #include "SoundSystem.h"
 
+#include "tinyxml.h"
```

Comment:
Why?

Reply:
This was from development and I forgot to remove it, sorry.